app/src/main/res/layout/activity_main.xml

```diff
@@ -29,6 +29,13 @@
             android:layout_height="0dp"
             android:layout_weight="1"/>
 
+        <FrameLayout
+            android:id="@+id/fragment_container_search"
```

Comment:
I imagine we would be using this fragment container for other screens which cannot be directly accessed from the tabs in the bottom navigation view, and would require the tabs to be hidden. This includes publish, the channel editor and the rewards views. We can give it a more generic id.

Reply:
There is `main_activity_other_fragment` which is used to show single content view. For all those usages, this container would be ok.

For publish, for example, if user is consuming some content and wishes to publish any new content, after clicking the Publish button, currently being played content would go to the mini player and the publish fragment would replace single content view on the same fragment container. The same for rewards or channel editor, as these are unrelated tasks.

I added the fragment container for search so user was able to click on results and see them on the `main_activity_other_fragment` at the same time, on tablets when the design is modified for large screens.